### ISSUE TITLE
Which hyperparameters to tune?

### ISSUE THREAD

**Issue by TBondat** [dbolya/yolact#246]
Hi,
as i'm running smaller dataset than the full coco + doing fine-tuning, it doesn't take so much time to get reasonable results. So i was thinking to maybe try to run like bayesian optimization to tune some hyperparameters of yolact.

Which hyperparameters do you advise me to release, to improve results? which ones are the most influent / interesting?
thanks

**Comment by dbolya** [comment]
Hmm interesting idea. Some things that are particularly influential are the anchor box scales and aspect ratios. Maybe also the number of anchor boxes. Apart from that, there's not much that you can really do a sweep over.

**Comment by TBondat** [comment]
ok nice, i keep that in mind, thanks!

**Comment by TBondat** [comment]
do you have the exact parameter names you mentionned? there are plenty of parameters available in config.py file :) 
thanks

**Comment by dbolya** [comment]
They are `pred_aspect_ratios` and `pred_scales`. Check `yolact_base_config` for details.

**Comment by abhigoku10** [comment]
@TBondat "run like bayesian optimization to tune some hyperparameters of yolact" can you please elaborate on this is there any different way to tune the hyperparams

**Comment by TBondat** [comment]
yes, you can also do basic grid search or random search for instance.
https://nanonets.com/blog/hyperparameter-optimization/
